README.md

```diff
@@ -8,7 +8,9 @@ The targeted use cases include certificate and key management for TLS authentica
 
 corePKCS11 is implemented on PKCS #11 v2.4.0, the full PKCS #11 standard can be found on the [oasis website](http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/os/pkcs11-base-v2.40-os.html).
```

Comment:
corePKCS11 is a library that implements a subset of the PKCS #11 v2.4.0 OASIS standard <link>, with a focus on operations involving asymmetric keys, random number generation, and hashing.